src/test/java/dev/openfga/sdk/api/client/OpenFgaClientTest.java

```diff
@@ -2380,4 +2466,42 @@ public void setAuthorizationModelId() throws Exception {
                 "OpenFgaClient.setAuthorizationModelId(String) is expected to persist its Authorization Model ID in its ClientConfiguration."
                         + "If this behavior ever changes, it could be a subtle breaking change.");
     }
+
+    private Matcher<String> isUUID() {
```

Comment:
It's weird that a method called isUUID returns a uuid

I would have expected it to return a boolean, whether the value is a uuid or not. Can we call it newUUID?

Reply:
For a public and non-test function I'd agree. In Java testing, this naming has been a convention of Hamcrest matchers for more than a decade now. The `Matcher` itself returns boolean for a given `String` in some Mock context https://hamcrest.org/JavaHamcrest/javadoc/2.2/org/hamcrest/Matchers.html

But reviewing the other matchers from Hamcrest, maybe it should be `anyUUID()` or something. I'll think on it a bit.

Also a minor nitpick to myself, it should be a static method.

Reply:
updated the name in https://github.com/openfga/java-sdk/pull/37/commits/23603940828703435405356892b57c14add3cfcb